pkg/volume/awsebs/aws_util.go

```diff
@@ -125,6 +126,20 @@ func (util *AWSDiskUtil) CreateVolume(c *awsElasticBlockStoreProvisioner, node *
 	return name, volumeOptions.CapacityGB, labels, fstype, nil
 }
 
+// getCandidateZones finds possible zones that a volume can be created in
+func getCandidateZones(cloud *aws.Cloud, selectedNode *v1.Node, pvcName string) (sets.String, error) {
```

Comment:
Why is `pvcName` passed as an argument?

Reply:
my bad, forgot to remove it during a refactor, changed the signature.